This build system writes the static library and executables directly
into the install prefix at compile time. Patch it to write to the
destdir instead.

This is a workaround for a badly-designed build system. The real fix
would be to remove these lines to let cmake put the compiled files into
the build directory as usual and then install them later but I'm leaving
it to the developers to retool the build system to do that.

https://github.com/irstlm-team/irstlm/issues/28
--- src/CMakeLists.txt.orig	2015-12-09 17:31:15.000000000 -0600
+++ src/CMakeLists.txt	2023-02-05 17:42:28.000000000 -0600
@@ -1,6 +1,6 @@
 # Set output directory
-SET(EXECUTABLE_OUTPUT_PATH ${CMAKE_INSTALL_PREFIX}/bin)
-SET(LIBRARY_OUTPUT_PATH ${CMAKE_INSTALL_PREFIX}/lib)
+SET(EXECUTABLE_OUTPUT_PATH @DESTROOT@${CMAKE_INSTALL_PREFIX}/bin)
+SET(LIBRARY_OUTPUT_PATH @DESTROOT@${CMAKE_INSTALL_PREFIX}/lib)
 
 ADD_DEFINITIONS("-D_LARGE_FILES")
 ADD_DEFINITIONS("-D_FILE_OFFSET_BITS=64")
